<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogWidget</class>
 <widget class="QWidget" name="LogWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LogWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
	
	font: 14pt &quot;微软雅黑&quot;;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QTextEdit" name="textEdit_details"/>
   </item>
   <item row="1" column="1">
    <widget class="QListView" name="listView_date"/>
   </item>
   <item row="1" column="0">
    <widget class="QListView" name="listView_month"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>月份:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>日期:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>内容:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
